src/custom-elements/src/components/dropdown/dropdown.tsx

```diff
@@ -233,27 +234,29 @@ export class Dropdown {
     return parseInt(getComputedStyle(this._dropdownMenuElement).getPropertyValue(`padding-${direction}`), 10);
   }
 
-  setMenuHeight() {
-    const menuItems = (this._dropdownMenuItemsWrapper ? this._dropdownMenuItemsWrapper.children : this._dropdownMenuElement.children) as HTMLAnchorElement[];
-
-    const itemsToShow = this.visibleItems
-      ? menuItems.length < this.visibleItems
-        ? menuItems.length
-        : this.visibleItems
-      : menuItems.length;
+  getTotalElementHeight(element: HTMLElement) {
+    return parseInt(getComputedStyle(element).getPropertyValue('height'), 10) + 
```

Comment:
As a sugestion, to avoid getting the computed style several times:

```js
    const computedStyle = getComputedStyle(element);

    return ['height', 'margin-top', 'margin-bottom'].reduce(
      (totalHeight, prop) => totalHeight + parseInt(computedStyle.getPropertyValue(prop), 10),
      0
    );
```